stepactions/eaas-get-ephemeral-cluster-credentials/0.1/eaas-get-ephemeral-cluster-credentials.yaml

```diff
@@ -26,11 +26,23 @@ spec:
   results:
     - name: kubeconfig
       description: Relative path to the kubeconfig in the mounted volume
+    - name: username
+      description: Relative path to the username file in the mounted volume
+    - name: password
```

Comment:
Just for the sake of avoiding heart attacks, can we rename this to something like "passwordPath"?

For consistency, `username` could also be renamed to `usernamePath` but it's probably ok to just record the username value directly in the result as that shouldn't be sensitive data.

Reply:
I updated it to `usernamePath` and `passwordPath`

Reply:
I think we should keep `username` instead of `usernamePath` and emit it directly in the results.

Reply:
updated

Reply:
@Kousalya1998 the only thing updated was the name of the param. The script is still writing this username to the credentials volume as the description states. The script should instead direct the value to the results.

Reply:
oh I didn't notice that it should emit it directly in the results. I think you want it to echo it in there